test/integration/get_triples_test.go

```diff
@@ -78,65 +87,102 @@ func TestGetTriples(t *testing.T) {
 			false,
 			-1,
 			nil,
+			PopObs,
+		},
+		{
+			[]string{
+				"dc/o/w2z8nx9y43k97",
+				"dc/o/mc1g2ew9yegq8",
+				"dc/o/28b93wpnlkjgc",
+				"dc/o/88cs3xqnmpp55",
+				"dc/o/23gt9k7fql176",
+				"dc/o/kyv7dxe4s18eh",
```

Comment:
nit: A note on what provenance / type of obs these are?

Reply:
Added statvar and mmethod as note